src/controllers/SafeNavigationBaseController.ts

```diff
@@ -69,6 +69,13 @@ export abstract class SafeNavigationBaseController<FORM> extends BaseController<
                 loggerInstance()
                     .info(`${SafeNavigationBaseController.name} - onGet: Application did not have navigation permissions to access ${this.httpContext.request.path}.`);
                 if (this.httpContext.request.path !== PENALTY_DETAILS_PAGE_URI) {
+
+                    // override access to Evidence Upload Page from Check your Appeal page, when skipped previously
+                    if (this.httpContext.request.path === EVIDENCE_UPLOAD_PAGE_URI &&
+                        this.httpContext.request.header('Referer')?.includes(CHECK_YOUR_APPEAL_PAGE_URI)) {
```

Comment:
What is this referer header?

Reply:
the referer header identifies the address of the previous webpage. Here i am checking if we are navigating to the  EVIDENCE_UPLOAD_PAGE_URI from the CHECK_YOUR_APPEAL_PAGE_URI page